src/styles/global.scss

```diff
@@ -77,7 +82,7 @@
   --page-padding-notebook: var(--space-5);
 
   // Target Size
-  --min-size: var(--space-7);
+  --min-size: 48px;
```

Comment:
Maybe not for this PR, but throwing it in here since the thought just came to me. 

I think we should rename this variable to be more specific, to something like **tap-min-size**. If I hadn't seen the comment above, I wouldn't know it's about the minimum size required for _tappable_/clickable elements on mobile.

Reply:
Agreed!

Reply:
Agreed!

Reply:
✅